transmit-lib/src/cmr/transmit/config.clj

```diff
@@ -92,6 +92,17 @@
 (def-app-conn-config urs {:port 3008})
 (def-app-conn-config virtual-product {:port 3009})
 
+(defconfig kms-scheme-override-json
+  "Allow for KMS schema urls to be overriden by setting this value to a JSON
+   strings such as:
+   {\"platforms\":\"file://frozen/platforms.csv\","
+    "\"mime-type\":\"mimetype?format=csv&version=special\"}
+    The reason this is needed is because CMR can only talk to the production KMS
+    server. SIT, UAT, and Production may all need to be changed to KMS schema
+    versions needed for these specific envirnments to overcome this limitation
+    and to present testing data as required."
+  "")
```

Comment:
The definition for `defmacro` states that the `:default` option is required. Additionally, maybe you can make use of a `:parser` here?

```
(defmacro defconfig
...
Options:

  * :default - Sets the default value if none is provided. Required.
  * :type - Sets the type of value the configuration parameter will have. The type will be used
  to determine how to parse the environment variable value.
  * :parser - A custom parser function to use to parse the environment variable value.
 ```

Reply:
o I missed the default part, and a I'll try the :parser setting.